### DISCUSSION TITLE
New .Net Maui project crashes when Sentry is added on iOS

### DISCUSSION THREAD

**Discussion by herczegzoltan** [getsentry/sentry-dotnet#3456] [Q&A] [answered]
Hi,

I have been developing an app and I tried to deploy on physical iOS but when the app starts always crashes I tried to figure out what is happening. 

In the end I removed Sentry  Nuget and the deployment was successful.

As confirmation that really Sentry was the issue I created a new maui project, successfully deployed to iOS then I added Sentry Nuget the app crashed at startup. 
```
 .UseSentry(options =>
        {
            options.Dsn = "myDsn";

        })
```
I only configured my dsn, It works on Android perfectly. Anyone has any idea whats happening?

**Comment by jamescrosswell** [comment, marked as answer]
Depending on whether you're running on a simulator or a physical device and which versions you're running, it could possibly be this:
- https://github.com/getsentry/sentry-dotnet/issues/3252
